packages/world/src/ObjectTypeLib.sol

```diff
@@ -118,6 +119,10 @@ library ObjectTypeLib {
     return self == ObjectTypes.AnyLog || self == ObjectTypes.AnyPlanks;
   }
 
+  function isFlora(ObjectTypeId self) internal view returns (bool) {
```

Comment:
Could we remove this? It'll be misleading cuz not all flora are pass through eg, leaves